io.github.loissascha.mangoverlay.yml

```diff
@@ -0,0 +1,24 @@
+app-id: io.github.loissascha.mangoverlay
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+command: mangoverlay
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=ipc
+  - --filesystem=host:ro
+  - --filesystem=home
+  - --talk-name=org.freedesktop.Flatpak
+modules:
+  - name: MangOverlay
+    sources:
+      - type: archive
+        url: https://github.com/loissascha/MangOverlay/releases/download/v0.2.9/mangoverlay-v0.2.9-linux-amd64.zip
```

Comment:
this does not work on aarch64. Did you see https://docs.flathub.org/docs/for-app-authors/requirements#flathubjson ?

Reply:
this seems go, why not build from source ? There are lots of go apps on Flathub.

Reply:
you have to allow network locally, using `build-args` generate the manifest, remove build-args and add the manifest.